tools/schemacode/setup.cfg

```
@@ -36,7 +36,7 @@ render =
     pandas
 tests =
     codecov
-    coverage<5.0
+    coverage[toml]<5.0
     flake8<5.0
```

While we're fiddling around here, can we try relaxing these? I suspect they were needed to wait for other tooling to catch up with an API-breaking change, but I hope we won't need them forever.

```suggestion
    coverage[toml]
    flake8
```